BraveShared/BraveStrings.swift

```diff
@@ -1595,13 +1601,25 @@ extension Strings {
         public static let introCardTitle = NSLocalizedString(
             "today.introCardTitle",
             bundle: .braveShared,
-            value: "Brave Today shows top stories in a completely private feed, just for you.",
+            value: "Today's top stories in a completely private feed, just for you.",
             comment: ""
         )
         public static let introCardBody = NSLocalizedString(
             "today.introCardBody",
             bundle: .braveShared,
-            value: "Your interests are matched on your device so your personal information never leaves your browser. New content updated throughout the day.",
+            value: "Brave Today is ad-supported with completely private and anonymized ads matched on your device.",
+            comment: ""
+        )
+        public static let introCardNew = NSLocalizedString(
+            "today.introCardNew",
+            bundle: .braveShared,
+            value: "New",
+            comment: ""
```

Comment:
Add comment what this `new` refers to, many languages use different words for it depending on context.
It should mention something like `new as new feature introduced`